Style_of_Pages.css

```diff
@@ -0,0 +1,97 @@
+html, body {
+    height: 100%;
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    display: flex;
+    flex-direction: column;
+    font-weight: 700;
+    color: black;
+    font-family: Arial, sans-serif;
+    background-image: url('https://i.pinimg.com/originals/fa/ba/63/faba6356d9a970b55b6658328e73d017.jpg');
+    background-size: cover;
+    background-position: center;
+}
+
+header {
+    background-color: #378b29;
+    background-image: linear-gradient(315deg, #378b29 0%, #74d680 74%);
+    color: black;
+    padding: 20px;
+    text-align: center;
+}
+
+main {
+    background-color: rgba(255, 255, 255, 0.6);
+    padding: 20px;
+    display: flex;
+    flex-direction: column;
+}
+
+.textbox {
+    background-color: rgba(255, 255, 255, 0.6);
+    border-radius: 10px;
+    padding: 20px;
+    margin-bottom: 20px;
+}
+
+nav ul {
+    list-style-type: none;
+    padding: 0;
+}
+
+nav ul li {
+    display: inline;
+    margin-right: 20px;
+}
+
+nav ul li a {
+    display: inline-block;
+    padding: 10px 20px;
+    text-decoration: none;
+    color: white;
+    border: 1px solid white;
+    border-radius: 5px;
+    margin: 0 10px;
+}
+
+.btn {
+    display: inline-block;
+    padding: 10px 20px;
+    background-color: transparent;
+    color: black;
+    text-decoration: none;
+    border: 2px solid black;
+    border-radius: 5px;
+    cursor: pointer;
+    transition: background-color 0.3s ease;
+}
+
+.btn:hover {
```

Comment:
some of the styles in these files are repeated across. consider setting up a global stylesheet to hold the styles that are shared across the various pages, and then additional individual stylesheets like you have set up for page-specific styles